examples/energy-management-app/esp32/sdkconfig.defaults

```diff
@@ -67,3 +67,7 @@ CONFIG_LWIP_IPV6_NUM_ADDRESSES=6
 
 # Enable sntp time sync
 CONFIG_ENABLE_SNTP_TIME_SYNC=y
+
+CONFIG_ENABLE_EXAMPLE_WATER_HEATER_DEVICE=y
+
+CONFIG_CHIP_DEVICE_CONFIG_ENABLE_WATER_HEATER_MANAGEMENT_TRIGGER=y
```

Comment:
Should DEM be turned on and EVSE triggers too?

Reply:
Not done?

Reply:
Code updated accordingly.